src/job/job.spec.js

```diff
@@ -167,4 +167,26 @@ describe("job", () => {
       ).rejects.toMatchSnapshot();
     });
   });
+
+  describe(".runSequence() :", () => {
+    afterEach(async () => {
+      await Promise.all(
+        xo.objects.all[config.vmIdXoTest].snapshots.map(id =>
+          xo
+            .call("vm.delete", { id, delete_disks: true })
+            .catch(error => console.error(error))
+        )
+      );
+    });
+
+    it("runs a job", async () => {
+      const id = await xo.createTempJob(defaultJob);
+      const snapshotsLength =
+        xo.objects.all[config.vmIdXoTest].snapshots.length;
+      await xo.call("job.runSequence", { idSequence: [id] });
+      expect(xo.objects.all[config.vmIdXoTest].snapshots.length).toBe(
```

Comment:
Please use `waitObjectState`  because we are waiting for a special VM state. And we don't know when we'll get the event.